tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/PackageCheckName.cs

```diff
@@ -0,0 +1,22 @@
+namespace Azure.Sdk.Tools.Cli.Models;
+
+/// <summary>
+/// Defines the types of validation checks that can be run on SDK packages.
+/// </summary>
+public enum PackageCheckName
+{
```

Comment:
```suggestion
public enum PackageCheckType
{
```

Reply:
(it's not a _name_, that I can tell?)

Reply:
I was leaning name because each checkName could contain different check types depending on the language (like formatCode is black for python but could be eslint or something for js) that might be overthinking it will circle back as I add more checks